src/applications/simple-forms/20-10207/config/form.js

```diff
@@ -305,8 +304,7 @@ const formConfig = {
         },
         nonVeteranMailingAddressPage: {
           depends: formData =>
-            formData.livingSituation.NONE &&
-            formData.mailingAddressYesNo &&
+            (formData.mailingAddressYesNo || formData.livingSituation.NONE) &&
```

Comment:
does veteranMailingAddressThirdPartyVeteranPage need to be updated as well? seems like a lot of these changes come in triples, but I only see 2 changes here.